ogc/display-ogc-api-collection/README.md

```diff
@@ -0,0 +1,43 @@
+# Display OGC API collection
+
+Display an OGC API feature collection and query features while navigating the map view.
+
+![Image of display OGC API collection](DisplayOGCAPICollection.png)
```

Comment:
I intentionally included the attribution here as I felt without some form of context the image looked a bit dull/messy? I've checked our image notes in the contribution wiki and it seems ok to include attribution: I'll leave this in for now but double check with the team in case there is a note somewhere else 👍